### PR TITLE
Updated metadata.json to adhere to proper syntax

### PR DESCRIPTION
**Pull request by bschonec**
"X"s are not allowed when using =, < or >.  

Also updated to reflect current puppetlabs-stdlib and puppetlabs-concat versions (which I've tested in my environment).

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -228,8 +228,9 @@
   $krb5_conf_owner          = 'root',
   $krb5_conf_group          = 'root',
   $krb5_conf_mode           = '0444',
-  $alter_etc_services       = false
-) {
+  $alter_etc_services       = false,
+  $includedir               = $mit_krb5::params::includedir,
+) inherits mit_krb5::params {
```
Comment (faxm0dem) [inline manifests/init.pp]: I'm afraid `inherits` is deprecated, and so is the `params.pp` paradigm.

File: manifests/init.pp [hunk 2]
```diff
@@ -266,18 +267,25 @@
...
-  
+
   if ($alter_etc_services == true) {
     class { '::mit_krb5::config::etc_services':
       require => Class['::mit_krb5::install']
     }
   }
 
   concat { $krb5_conf_path:
-    owner  => $krb5_conf_owner,
```
Comment (faxm0dem) [inline manifests/init.pp]: please separate code changes and cosmetic changes to different commits

**Comment by faxm0dem** [comment]
thanks for the changes!
I added a few comments in-line. Hopefully @riton will add some more

File: manifests/init.pp (hunk 1 above)
Comment (bschonec) [reply manifests/init.pp]: Obviously I'm broadcasting my lack of skill with GIT.  It was not my intention to include the inherits/params.pp design in my pull request last month.  I made the hackish inherits/params.pp code yesterday and didn't realize it'd get included in my pull request from last month.   

I tried copying the format/syntax in the current ../templates/*erb files but I'm no Ruby programmer and the logic behind the code escapes me.

**Comment by faxm0dem** [comment]
Am I right that by saying that your only change for this PR is the requirements in metadata.json?
If that's correct, I can happily submit a PR to your branch ;-)

**Comment by bschonec** [comment]
Yep.  Only the metadata.json file needs updating so that librarian-puppet and R10k are happy.

**Comment by faxm0dem** [comment]
okay then, just do the following in order to:

0. back up your current folder for extra safety
1. Make a new branch with your work in progress
2. Reset your master branch to origin
3. Create a new branch for the metadata change
4. Open a new PR for that branch

```
# 1.
git co -b work_in_progress
# 2.
git co master
git fetch origin
git reset --hard origin/master
# 3.
git co -b fix/metadata
git cherry-pick f44a66ca0fbde477d72df63bcfd44da45c123b5c
git push -u origin fix/metadata
```

**Comment by riton** [comment]
The latest release `v0.4.0` should fix thix issue. Thanks for the merge request. Feel free to open another issue / pull request if you find something that does not suit your needs.

**Pull request closed without merging**